src/AutoMapper.php

```diff
@@ -145,6 +146,9 @@ public function getMetadata(string $source, string $target): ?MapperGeneratorMet
         return $this->metadata[$source][$target];
     }
 
+    /**
+     * @deprecated since 8.1, will be removed in 9.0.
+     */
```

Comment:
Would require a `trigger_deprecation` call to inform people to not use it anymore and what is the replacement.